### PR TITLE
RTEMIS-137: Added the markup texts.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-137

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Added the markup texts.
- Modified the parent feild to radios.

---
### Extras
`(Add Screenshots or videos)`
image
image

### CODE REVIEW

File: modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php [hunk 1]
```diff
@@ -31,12 +31,16 @@ public function buildForm(array $form, FormStateInterface $form_state) {
...
       $form['#attributes']['id'] = 'school-detail-wrapper';
+
       $form['field_school_preference_wrapper'] = [
         '#type' => 'container',
         '#attributes' => [
           'id' => 'school-preference-wrapper',
         ],
       ];
+      $form['field_school_preference_wrapper']['selection_markup'] = [
+        '#markup' => '<div class="selection-detail-markup"><p>Select Nearest school within 1 km from your residence first. You can select more than one.</p></div>',
```
Comment (jaswinsingh) [inline modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php:42]: Please wrap the text with `t()` method

File: modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php [hunk 2]
```diff
@@ -95,6 +99,10 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         ],
       ];
 
+      $form['field_parent_type']['parent_details_markup'] = [
+        '#markup' => '<div class="parent-detail-markup"><p>Detailed Father/Mother/Guardian(It is mandatory to fill the details of atleast one)</p></div>',
```
Comment (jaswinsingh) [inline modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php:103]: Please wrap the text with `t()` method

Comment (RohanSingh-inno) [reply modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php]: ok

File: modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php (hunk 1 above)
Comment (RohanSingh-inno) [reply modules/rte_mis_student/src/Form/OverrideMiniNodeForm.php]: ok

**Review by jaswinsingh — APPROVED** [review]

**Pull request merged**